### DISCUSSION TITLE
Marlin Legacy or 2?

### DISCUSSION THREAD

**Discussion by Pridanc** [mriscoc/Ender3V2S1#1299] [General]
Hey everyone. Although I tend to use CURA because that was my first slicer, I've started to us ORCA for some other things.  Along the way ORCA is asking which version of Marlin I use.   The default here  is  "Marlin Legacy" but there is an option for "Marlin 2".  Which should I use or does it matter? My Ender has a new SKR Mini 3 v3 board and flashed with Mriscoc as my firmware.

**Comment by mriscoc** [comment]
The Professional Firmware is based on the latest Marlin.

**Comment by Pridanc** [comment]
​Price Cobb
Confidentiality
Statement & Notice: This email is covered by the Electronic Communications
Privacy Act, 18 U.S.C. 2510-2521 and intended only for the use of the
individual or entity to whom it is addressed. Any review, retransmission,
dissemination to unauthorized persons or other use of the original message and
any attachments is strictly prohibited. If you received this electronic
transmission in error, please reply to the above-referenced sender about the
error and permanently delete this message. Thank you for your cooperation.

Sent from Proton Mail Android

-------- Original Message --------
